xml/System/Environment.xml

```diff
@@ -1753,8 +1753,8 @@ To identify the operating system platform, for example, Linux or Windows, you ca
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the number of processors on the current machine.</summary>
-        <value>The 32-bit signed integer that specifies the number of processors on the current machine. There is no default. If the current machine contains multiple processor groups, this property returns the number of logical processors that are available for use by the common language runtime (CLR).</value>
+        <summary>Gets the number of processors on the current machine or container (like a Docker container).</summary>
```

Comment:
This description is too constraining and does not include process affinity or non-container Windows job objects. Maybe it should read "Gets the number of processors available for the current process"?  We could add that in a non-restricted environment it is the same as the number of processors on the current machine.

Also, is it possible to incorporate at least some of Jan's suggestions from https://github.com/dotnet/runtime/issues/13691#issuecomment-548464190?